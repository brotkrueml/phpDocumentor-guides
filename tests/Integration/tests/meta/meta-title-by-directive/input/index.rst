
.. title:: Title from directive

==============
Document Title
==============

Lorem Ipsum Dolor.
