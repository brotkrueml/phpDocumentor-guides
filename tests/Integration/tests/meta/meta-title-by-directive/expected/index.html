<!DOCTYPE html>
<html lang="en">
<head>
<title>Title from directive</title>
</head>
<body>
    <div class="section" id="document-title">
    <h1>Document Title</h1>
        <p>Lorem Ipsum Dolor.</p>
    </div>
</body>
</html>
